### DISCUSSION TITLE
MetaMask option is duplicated in WalletConnect modal v3, when it's both in `featuredWalletIds` and installed

### DISCUSSION THREAD

**Discussion by SergeyVolynkin** [WalletConnect/walletconnect-monorepo#3874] [Developer Support]
Hi, Wallet Connect team!

Here is our prod WalletConnect usage:
<img width="515" alt="image" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/34238697/30c0e87d-f8ea-459f-bc17-168f400b126b">

`Question 1`:  is it a bug that MetaMask wallet is duplicated (First one(the "installed"-tagged one) is automatically added by WalletConnect,another one comes from `featuredWalletIds` we've set). It may make sense to display only one wallet even if `featuredWalletIds` is set to that wallet

`Question 2`: Is possible to strictly configure which wallets are shown when user opens WalletConnect modal v3
Is it possible to configure so that only 3 options are shown?:
  – MetaMask
  – Fireblocks
  – All Wallets

`Question 3`: Shouldn't "Terms of Service" and "Privacy Policy" be of color `--w3m-accent` when it is set in `theme`? (we've `--w3m-accent` to purple, but links are not highlighted)

**Comment by glitch-txs** [comment]
Such a beautiful issue, we'll get it fixed.

Question 2: Do you mean to remove Browser Wallets and All Wallets button?

Question 3: we can add that, could you open an issue on Web3Modal repo as feture request?

**Comment by SergeyVolynkin** [comment]
> Question 2: Do you mean to remove Browser Wallets and All Wallets button?

Sorry if unclear, @glitch-txs  I meant if it is possible / will be possible to configure into this:
<img width="362" alt="image" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/34238697/8f034f09-ff03-4807-88fb-1c32f6681707">

> Question 3: we can add that, could you open an issue on Web3Modal repo as feture request?

Yes, will do

**Reply by glitch-txs** [reply]
could you open an issue for this as well? I've opened one for the duplicated, this is not possible at the moment

**Reply by SergeyVolynkin** [reply]
Yep, thank you  @glitch-txs, I will open another issue tomorrow

**Reply by SergeyVolynkin** [reply]
Created https://github.com/WalletConnect/web3modal/issues/1449, @glitch-txs

**Comment by SergeyVolynkin** [comment]
opened: https://github.com/WalletConnect/web3modal/issues/1447
